tool/BUILD

```diff
@@ -14,4 +14,10 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
\ No newline at end of file
+#
+
+py_library(
+    name = "common",
+    srcs = ["common.py"],
+    visibility = ["//visibility:public"]
+)
```

Comment:
new line

Reply:
I think this should sit in its own package

Reply:
> new line

Solved in https://github.com/graknlabs/dependencies/pull/136/commits/e44a270ac97b4d28e56e409d91fd75663ccb164c

> I think this should sit in its own package

Solved in https://github.com/graknlabs/dependencies/pull/136/commits/7edfbf380aead527fb239423a2f51b3a6b0d6db8